receiver/prometheusreceiver/internal/transaction_test.go

```diff
@@ -705,6 +1067,11 @@ func TestMetricBuilderHistogram(t *testing.T) {
 				pt2.SetTimestamp(tsNanos)
 				pt2.SetStartTimestamp(startTimestamp)
 				pt2.Attributes().PutString("foo", "bar")
+				e2 := pt2.Exemplars().AppendEmpty()
+				e2.SetDoubleVal(2)
+				e2.SetTimestamp(pcommon.NewTimestampFromTime(time.UnixMilli(1663113420863)))
```

Comment:
use the function I mentioned to you. I think I commented before about this, please fix everywhere where my comment applies.

Reply:
Done